File: Web/alisha0903_AlishaRaghav_2125cse1189_02/FunctionAndArrays/spanOfArray.cpp
#include<iostream>
using namespace std;

int main(){
    
    int n,i,max,min,d;
    cin>>n;
    int a[n];
    for(i=0;i<n;i++)
    cin>>a[i];
    max=a[0];
    min=a[0];
    for(i=0;i<n;i++)
    {
        if(a[i]>max)
        max=a[i];
        if(a[i]<min)
        min=a[i];
    }
    d=max-min;
    cout<<d;
    return 0;
    
    
}

File: Web/Tushar-sahani_TusharSahani_2125cs1072_2/FunctionAndArrays/SpanOfArray.java
package coderspear.FunctionAndArrays;
import java.util.*;
public class SpanOfArray {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int[] arr = new int[n];
        for(int i=0;i<n;i++){
            arr[i] = sc.nextInt();
        }
        int max = 0,min = arr[0];
        for(int i = 0;i<n;i++){
            if(max<arr[i]){
                max = arr[i];
            }
            if(min>arr[i]){
                min = arr[i];
            }
        }
        System.out.println(max - min);
     }
}
